### DISCUSSION TITLE
Choosing motifs for motif plots

### DISCUSSION THREAD

**Discussion by coopershawna** [stuart-lab/signac#819] [Q&A]
Is there a way to either feed in a list of motifs of interest that is a subset of an "enriched.motif" or choose beyond the top 6 using something besides "head" or "tail"? For example, I ran the enriched motif analysis and I would like to show motifs that are beyond the first 6 listed.  Sometimes the most biologically interesting (and still highly significant TF motifs are actually number 10 on the list. I made a list and tried to feed it in the same way I can for doing heatmaps, ect and it kept failing (even using the motif ID or name).  

Thank you for your time and assistance.

Shawna

**Comment by timoast** [comment]
The `motifs` argument in `MotifPlot()` accepts a vector of motif IDs to plot. Can you show the code you're running that's not working for you?
